min-image/runtimes/python/ol.c

```diff
@@ -388,6 +454,8 @@ static PyMethodDef OlMethods[] = {
                                   {"unshare", (PyCFunction)ol_unshare, METH_NOARGS, "unshare"},
                                   {"fork", (PyCFunction)ol_fork, METH_NOARGS, "fork"},
                                   {"enable_seccomp", (PyCFunction)ol_enable_seccomp, METH_NOARGS, "enable_seccomp"},
+                                  {"modify_cap", (PyCFunction)ol_modify_cap, METH_VARARGS, "modify_cap"},
```

Comment:
@tylerharter This is still a work in progress. But the basic idea (based on my understanding) was that maybe we can expose a function here to modify/ drop capabilities. I'm still not sure which capabilities should be dropped and which ones should be kept?